README.md

```diff
@@ -1,72 +1,41 @@
 # Microsoft Azure Active Directory Authentication Library (ADAL) for Python
+=====================================
 
-The ADAL for python library makes it easy for python applications to authenticate to AAD in order to access AAD protected web resources.
+[![Build Status](https://travis-ci.org/AzureAD/azure-activedirectory-library-for-python.svg?branch=dev)](https://travis-ci.org/AzureAD/azure-activedirectory-library-for-python)
+[![Documentation Status](https://readthedocs.org/projects/adal-python/badge/?version=latest)](https://adal-python.readthedocs.io/en/latest/?badge=latest)
 
-## Usage
+|[Getting Started](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki)| [Docs](https://aka.ms/aaddev)| [Samples](https://github.com/azure-samples?query=active-directory)| [Support](README.md#community-help-and-support)
+| --- | --- | --- | --- |
 
-### Install
 
-To support 'service principal' with certificate, ADAL depends on the 'cryptography' package. For smooth installation, some suggestions:
+The ADAL for Python library enables python applications to authenticate with Azure AD and get tokens to access Azure AD protected web resources.
 
-* For Windows and macOS
+You can learn in detail about ADAL Python functionality and usage documented in the [Wiki](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki).
 
-  Upgrade to the latest pip (8.1.2 as of June 2016) and just do `pip install adal`.
+## Versions
+Current version - 0.6.0
```

Comment:
I would suggest to not mention "current version" here, especially when we are already going to provide the link to the Release history page below. Customers can do a one-click and know the single-source-of-truth about the latest (and, all) version information.

The downside of mentioning it here is it introduces one more place for us to update during future releases. If we could avoid that, then we probably should. :-)

Reply:
I was trying to match other SDKs but I agree with your suggestion of avoiding the duplication.